helpers/README.md

```diff
@@ -77,6 +77,17 @@ requestIdleCallback((deadline) => {
 }, { timeout: 1000 });
 ```
 
+## queueMicrotask
+
+A polyfill for [queueMicrotask](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/queueMicrotask).
```

Comment:
Looks good.  The only think I might add is a link to an article that provides a bit more info on microtasks (such as the [mozilla one](https://developer.mozilla.org/en-US/docs/Web/API/HTML_DOM_API/Microtask_guide)). Not a big deal and devs can go searching if they are curious to know more, but for most, they are this mysterious thing that presenters mention at conferences.